vignettes/MSnbase-centroiding.Rmd

```diff
@@ -209,26 +222,27 @@ abline(h = prol_mz, col = "red", lty = 2)
 
 ```
 
-The smoothed centroided data for proline still show considerable
-fluctuations of m/z values and poor chromatographic data.
+The smoothed centroided data for proline still show deviation
+fluctuations of m/z values as well as poor chromatographic data.
```

Comment:
shouldn't it read *... still show deviation of m/z values...* instead of *...still show deviation fluctuations of m/z values...*?

Reply:
Fixed, thanks.